packages/ui/PricingTable/index.tsx

```diff
@@ -1,7 +1,29 @@
-import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@formbricks/ui/Tooltip";
 import { CheckIcon, XMarkIcon } from "@heroicons/react/24/outline";
+import { TooltipProvider, Tooltip, TooltipTrigger, TooltipContent } from "@radix-ui/react-tooltip";
 
-export const PricingTable = ({ leadRow, pricing, endRow }) => {
+interface SpecialRow {
+  title: string | JSX.Element;
+  free: string | JSX.Element;
+  paid: string | JSX.Element;
+}
+
+interface PricingRow {
+  name: string;
+  free: string | boolean;
```

Comment:
how can it be both?

Reply:
as discussed, moving this back to UI for now